iosApp/iosApp/Assets.xcassets/Google-Icon.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "google.jpg",
```

Comment:
@nasir-momin Can we use an SVG instead of a jpg?
https://drive.google.com/file/d/1js3-O7I_dv70rVvlI7104h2YA5offBaY/view?usp=sharing

Reply:
Yeah sure in future I ll start using sag format,

 but this icon is to be removed in future, Since Sign In with google has completely different type of implementation 

https://medium.com/@nasirahmedmomin/integrate-google-signin-in-swiftui-ios-b227f5b243b2